src/components/CheckBox/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import styled from 'styled-components';
+import checkMark from '../../assets/properties/displayProperties/icons/checkmark.svg';
```

Comment:
This asset doesn't really belong under `displayProperties` - Move it to the CheckBox folder along with the other related files

Reply:
Done.